contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveDrillNativeScanBatchCreator.java

```diff
@@ -123,6 +123,7 @@ public ScanBatch getBatch(FragmentContext context, HiveDrillNativeParquetSubScan
           // in the first row group
           ParquetReaderUtility.DateCorruptionStatus containsCorruptDates =
               ParquetReaderUtility.detectCorruptDates(parquetMetadata, config.getColumns(), true);
+          logger.info(containsCorruptDates.toString());
```

Comment:
not a very useful logging statement to check in, should at least contain a message about what this value is. This will just print a boolean value. If it was just for local debugging it can be removed.